addons/print/wizard/print_order_wizard.py

```diff
@@ -0,0 +1,143 @@
+    @api.onchange('provider_id')
+    def _onchange_provider_id(self):
+        self.provider_balance = self.provider_id.balance
+        self.provider_environment = self.provider_id.environment
+
+    # --------------------------------------------------
+    # METHODS
+    # --------------------------------------------------
+    @api.model
+    def default_get(self, fields):
+        """ create the lines on the wizard """
+        res = super(print_order_wizard, self).default_get(fields)
+
+        active_ids = self._context.get('active_ids', [])
+        active_model = self._context.get('active_model', False)
+
+        if active_ids and active_model:
+            # create order lines
+            lines = []
+            for rec in self.env[active_model].browse(active_ids):
+                lines.append((0, 0, {
+                    'res_id': rec.id,
+                    'res_model': active_model,
+                    'partner_id' : rec.partner_id.id,
+                    'last_send_date' : rec.print_sent_date,
+                }))
+            res['print_order_line_wizard_ids'] = lines
+        return res
+
+    @api.multi
+    def action_apply(self):
+        print "action_apply"
+        Print_order = self.env['print.order']
+        for wizard in self:
+            # raise warning if the provider is not configured
+            wizard.provider_id.check_configuration()
+            # don't do anything if the balance is too small in the production mode (in test mode, allow all)
+            if wizard.provider_balance <= 0.0 and wizard.provider_environment == 'production':
+                raise Warning(_('Your credit provider is too small. Please, configure your account (> Settings > Print Provider), and put some credit on your account, since you are in Production Mode.'))
+            for line in wizard.print_order_line_wizard_ids:
+                Print_order.create({
+                    'ink' : wizard.ink,
+                    'paper_weight' : wizard.paper_weight,
+                    'provider_id' : wizard.provider_id.id,
+                    'currency_id' : wizard.currency_id.id,
+                    'user_id' : self._uid,
+                    'res_id' : line.res_id,
+                    'res_model' : line.res_model,
+                    # duplicate partner infos
+                    'partner_id' : line.partner_id.id,
+                    'partner_name' : line.partner_id.name,
+                    'partner_street' : line.partner_id.street,
+                    'partner_street2' : line.partner_id.street2,
+                    'partner_state_id' : line.partner_id.state_id.id,
+                    'partner_zip' : line.partner_id.zip,
+                    'partner_city' : line.partner_id.city,
+                    'partner_country_id' : line.partner_id.country_id.id,
+                })
+        return {'type': 'ir.actions.act_window_close'}
+
+
+class print_order_line_wizard(models.TransientModel):
+
+    _name = 'print.order.line.wizard'
```

Comment:
Here the line has a meaning, because it will create new print orders after the wizard, if I understood well.